package.json

```diff
@@ -297,6 +251,57 @@
             "markdownDescription": "Specifies whether to see only net new issues or known issues. Only applies to Code Security and Code Quality.\n\nNote: this is an experimental feature. Please reach out to [support.snyk.io](https://support.snyk.io) for more details."
           }
         }
+      },
+      {
+        "id": "onboarding",
```

Comment:
I don't like `Onboarding`. I'd chuck the welcome notification setting and just go for a category `Folders` and put Trust, Folderconfig and all paths there. @neilm-snyk , WDYT?

Reply:
renamed to initialization